model/trace/instrumentation/graphql.yml

```diff
@@ -24,9 +24,63 @@ groups:
             - id: subscription
               value: "subscription"
               brief: "GraphQL subscription"
-        examples: ['query', 'mutation', 'subscription']
-      - id: document
-        brief: "The GraphQL document being executed."
+        examples: ["query", "mutation", "subscription"]
+      - id: document.content
+        brief: "The content of the GraphQL document being executed."
         type: string
         note: The value may be sanitized to exclude sensitive information.
-        examples: 'query findBookById { bookById(id: ?) { name } }'
+        examples: "query findBookById { bookById(id: ?) { name } }"
+      - id: document.sha256
+        brief: "The SHA256 hash of the document.content"
+        type: string
+        examples: "400483f38c08e8a3d3b972409c9dfb8e4a326e1b1940864932acd9f873d8664c"
+      - id: document.id
+        brief: >
+          The id of the document beeing executed. This is a hash of the document provided by the user to identitfy 
+          persisted queries.
+        type: string
+        examples: "aa3e37c1bf54708e93f12c137afba004"
+      - id: error.count
+        brief: "The number of errors that occurred during the operation."
+        type: int
+        examples: 3
+    # TODO should we have something like outcome (success, failure)
+    # TODO how do we specify errors?
+    # TODO Should we ref network.transport, network.type, server.address etc?
+    # TODO There are more spans like validation, parsing, variable coercion and response formatting. Should we add them as separate span types?
```

Comment:
It provides a lot of value to know how long a specific operation is spending validating, parsing or formatting. 
Should we add specific spans for this?

Reply:
We can model how the trace should like as we want - One thing to always keep in mind is verbosity. Spans costs money in the end. Creating too many may be a problem in some cases, so if it's not essential, maybe they can be turned off or opt-in. 

If so, please open a separated issue explaining the value/what it should do so we can tackle it in a separate PR.